bigquery_etl/dryrun.py

```diff
@@ -472,12 +476,12 @@ def is_valid(self):
             return False
 
         if self.dry_run_result["valid"]:
-            print(f"{self.sqlfile!s:59} OK")
+            print(f"{self.sqlfile!s:59} OK, took {self.dry_run_duration or 0:.2f}s")
         elif self.get_error() == Errors.READ_ONLY:
             # We want the dryrun service to only have read permissions, so
             # we expect CREATE VIEW and CREATE TABLE to throw specific
             # exceptions.
-            print(f"{self.sqlfile!s:59} OK")
+            print(f"{self.sqlfile!s:59} OK, DDL/DML skipped")
```

Comment:
To match the format of the other error-but-OK message.
```suggestion
            print(f"{self.sqlfile!s:59} OK but DDL/DML skipped")
```